libs/common/src/dtos/activity.dto.ts

```diff
@@ -81,6 +81,10 @@ export class LocationDto {
   @IsOptional()
   @IsEnum(UnitTypeDto)
   units?: UnitTypeDto;
+
+  @IsOptional()
+  @IsString()
+  type?: string = 'Place';
```

Comment:
So these were supposed to only be request api dtos (I think my vague naming might be the reason of confusion here) and I wouldn't try to use them for activity dtos. 
Some of the parameters like this one that could get hardcoded was eliminated from the request dtos. I would recommend that if you are going to create ActivityContent dtos keep them separate from these ones.

the best case scenario would be if we can use activity content library but if that is hard to achieve then duplicating those would be more preferable that changing these. I think we should be able to create all activity content from existing requests but please let me know if you are think any value is missing.

Reply:
This was probably due to conflict merges, I will remove this as its not being used